File: sdks/csharp/src/NovaBilling/Auth/Requests/RefreshTokenDto.cs
using System.Text.Json.Serialization;
using NovaBilling.Core;

namespace NovaBilling;

[Serializable]
public record RefreshTokenDto
{
    /// <summary>
    /// Refresh token
    /// </summary>
    [JsonPropertyName("refreshToken")]
    public required string RefreshToken { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: sdks/php/src/Auth/Requests/RefreshTokenDto.php
<?php

namespace NovaBilling\Auth\Requests;

use NovaBilling\Core\Json\JsonSerializableType;
use NovaBilling\Core\Json\JsonProperty;

class RefreshTokenDto extends JsonSerializableType
{
    /**
     * @var string $refreshToken Refresh token
     */
    #[JsonProperty('refreshToken')]
    public string $refreshToken;

    /**
     * @param array{
     *   refreshToken: string,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->refreshToken = $values['refreshToken'];
    }
}
